SimpactApp/company/dashboard.html

```diff
@@ -0,0 +1,329 @@
+<!doctype html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta charset="UTF-8">
+	<link rel="stylesheet" type="text/css" href="../template.css">
+
+	<title> Template Dashboard </title>
```

Comment:
Should we change it from Template Dashboard to something else?

Reply:
I deleted this file from my local device don't know why it uploaded!

Reply:
Can you please delete it once again? It may be a merge artifact - by pulling the master in your branch you may have brought it back.

`git rm SimpactApp/company/dashboard.html` (on your branch, followed by commit and push) should do the trick.